src/OrbitTest/OrbitTest.cpp

```diff
@@ -24,7 +24,8 @@
 // removed once we split the introspection and manual instrumentation apis TODO(b/182848566).
 namespace orbit_api_wrapper {
 #include "Api/Orbit.h"
-}
+orbit_api_v0 g_orbit_api_v0;
```

Comment:
does it work as expected if this is inside namespace? I guess compiler ignores namespace in this case because of extern "C" in the header. what does `readelf -d` say?

Reply:
`readelf -d` doesn't return anything, but 'readelf -s' gives me:

`736: 0000000000068110    96 OBJECT  GLOBAL DEFAULT   29 g_orbit_api_v0
`